Updated RC doc for enabling RC on agent by default (#19286)
DataDog/documentation: content/en/agent/remote_config/_index.md

1. You configure select product features in the Datadog UI.
2. The product feature configurations are securely stored within Datadog.

> 3. Agents in your environments securely poll, receive, and automatically apply configuration updates from Datadog. Tracing libraries, deployed in your environments, communicate with Agents to request and receive configuration updates from Datadog.

Reviewer: ```suggestion
3. Agents in your environments securely poll, receive, and automatically apply configuration updates from Datadog.
4. Tracing libraries, deployed in your environments, communicate with Agents to request and receive configuration updates from Datadog.
```

**Note**: Configuration changes applied through Remote Configuration are not shown in your Agent configuration file.

> ## Support products and feature capabilities

Reviewer: ```suggestion
## Supported products and feature capabilities
```

### Prerequisites

- Datadog Agent version `7.41.1`  (`7.42.0` for APM sampling rate, `7.43.0` for APM Remote Instrumentation) or higher installed on your hosts or containers. 

> - For Datadog products that use tracing libraries, you also need to upgrade your tracing libraries to a Remote Configuration compatible version. For ASM Protection capabilities and ASM 1-click activation, see [Compatibility Requirements][6]. For Dynamic Instrumentation, see [Compatibility Requirements][20].

Reviewer: ```suggestion
- For Datadog products that use tracing libraries, you also need to upgrade your tracing libraries to a Remote Configuration-compatible version. For ASM Protection capabilities and ASM 1-click activation, see [ASM compatibility requirements][6]. For Dynamic Instrumentation, see [Dynamic Instrumentation prerequisites][20].
```

3. On the [Remote Configuration][8] page, enable Remote Configuration. This enables Datadog components across your organization to receive configurations from Datadog.

4. Select an existing API key or create a new API key, and enable the Remote Configuration capability on the key:

   {{<img src="agent/remote_config/RC_Key_updated.png" alt="API Key properties with Remote Configuration capability Enable button." width="90%" style="center">}}

5. Update your Agent configuration file:

> Note: This step is required only for Agent versions 7.46.0 or lower. Starting with Agent version 7.47.0, remote_configuration.enabled is set to true by default in the Agent.

Reviewer: ```suggestion
**Note:** This step is required only for Agent versions 7.46.0 or lower. Starting with Agent version 7.47.0, `remote_configuration.enabled` is set to `true` by default in the Agent.
```

  | DISABLED       |   The Agent deployed in your environment has `remote_config.enabled` set to false in its `datadog.yaml` configuration file. Set `remote_config.enabled` to true if you want to enable Remote Configuration on the Agent. This status displays when the Agent version is `7.45.0` or higher. | 
  | NOT CONNECTED       | The Agent cannot be found in the Remote Configuration service and could have `remote_config.enabled` set to true or false in its `datadog.yaml` configuration file. Check your local Agent configuration or your proxy settings. This status displays when the Agent version is higher than `7.41.1` but lower than `7.45.0`.            | 
  | UNSUPPORTED AGENT   | The Agent is on a version that is not Remote Configuration capable. To fix this issue, update the Agent to the latest available version. |

## Opting out of Remote Configuration at the Agent level

> Starting with Agent version 7.47.0, remote_configuration.enabled is set to true by default in the Agent. This will cause the agent to request configuration updates from the Datadog site. To receive configurations from Datadog, you still need to enable Remote Configuration at the organization level and enable Remote Configuration capability on your API Key from the Datadog UI, and allow outbound HTTPS access to Remote Configuration [endpoints][17] from your environment. If you don't want your agent to send configuration requests to Datadog, you can set remote_configuration.enabled to false in the Agent.

Reviewer: ```suggestion
Starting with Agent version 7.47.0, `remote_configuration.enabled` is set to `true` by default in the Agent. This setting causes the Agent to request configuration updates from the Datadog site.

To receive configurations from Datadog, you need to take the following steps:
- Enable Remote Configuration at the organization level.
- Enable Remote Configuration capability on your API Key from the Datadog UI.
- Allow outbound HTTPS access to Remote Configuration [endpoints][17] from your environment.

If you don't want your Agent to send configuration requests to Datadog, you can set `remote_configuration.enabled` to `false` in the Agent.
```

Author: Can we add "also" to the sentence? Reason being we want to state that in addition to setting RC to true in the agent, customers also need to take additional steps to fully enable RC. 

To receive configurations from Datadog, you also need to take the following steps:

{{< tabs >}}
{{% tab "Configuration YAML file" %}}

> Change remote_configuration.enabled from true to false in your configuration YAML file:

Reviewer: ```suggestion
Change `remote_configuration.enabled` from `true` to `false` in your configuration YAML file:
```

> Change remote_configuration.enabled from true to false in your configuration YAML file:

Reviewer: Can you add any more information about the configuration file? What the filename is, and where it is located, perhaps?

Author: The filename is datadog.yaml and location varies. We have a link in our page here:
https://docs.datadoghq.com/agent/guide/agent-configuration-files/?tab=agentv6v7#agent-main-configuration-file
I will link to it.
